docs/serverless-demo.md

```diff
@@ -1,28 +1,26 @@
-# Welcome to the Apollo Showcase Demo
+# Welcome to the **{{ graph.name }}** API! 🎉
 
-Welcome to the **{{ graph.name }}** API! 🎉 You can follow along in this README to walk through a guided demo or just explore yourself.
+You can follow along in this README to walk through a guided demo or just explore around yourself.
 
-> If you're new to Apollo, learn more about GraphOS in [our docs](https://www.apollographql.com/docs/studio/).
```

Comment:
Removed so that they stay on the README page instead of getting distracted by the docs link.